sessions/css-positioning/demo-end/index.html

```diff
@@ -6,215 +6,119 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>HTML & CSS</title>
+    <title>CSS Positioning</title>
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
     <header class="header">
-      <h1 class="header__heading">CSS positioning</h1>
+      <h1 class="header__heading">
+        Sweeties<span class="plate" role="img" aria-label="A plate">🍽</span>
+      </h1>
     </header>
-    <main>
-      <section class="disclaimer">
-        <h2 class="disclaimer__title">disclaimer</h2>
-        <div class="disclaimer__icon">!</div>
-        <p class="disclaimer__text">
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Alias vitae
-          libero harum! Odit consequatur veritatis, hic qui, facere distincLorem
-          ipsum dolor sit, veritatis, hic qui, facere distincorem ipsum dolor
-          sit, amet consectetur adipisicing elit. Alias vitae libero harum! Odit
-          consequatur veritatis, hic qui, facere distinc
+
+    <main class="main">
```

Comment:
```suggestion
    <main class="content">
```
Maybe a more descriptive class name helps to understand better why BEM is useful.

Reply:
I'm indifferent, if you think this should be changed, I'll change it everywhere. :)

Reply:
I feel like "content" is a bit more descriptive about the Block.
I don't know if you know this little trick,I just learned this during my Research time, so i want to share: If you right-click on the class name in the HTML file and click on "Rename Symbol" every actually related (equal) name will be renamed. Its not renaming the string, which is super useful :)